moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -361,5 +375,7 @@ class TrajectoryExecutionManager
   double allowed_start_tolerance_;  // joint tolerance for validate(): radians for revolute joints
   double execution_velocity_scaling_;
   bool wait_for_trajectory_completion_;
+
+  std::shared_ptr<rclcpp::SyncParametersClient> manage_controllers_parameters;
```

Comment:
I think this is not necessary for the parameter lookup, since we can use `rclcpp::Node` parameter API. But it is needed if we decide to use parameter event replace dynamic reconfig. Temporarily removed here.